client/components/web-preview/component.jsx

```diff
@@ -90,13 +86,6 @@ export class WebPreviewModal extends Component {
 		this.setDeviceViewport = this.setDeviceViewport.bind( this );
 	}
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillMount() {
-		// Cache touch and mobile detection for the entire lifecycle of the component
-		this._hasTouch = hasTouch();
```

Comment:
I don't see a good reason to cache `hasTouch()`. We don't cache it in any of the other instances throughout Calypso.